.ci/Jenkinsfile-SITL_tests_ASan

```diff
@@ -61,6 +61,13 @@ pipeline {
               vehicle: "iris"
             ],
 
+            [
+              name: "Rover 1",
```

Comment:
@dagar what was your thought behind "Rover 1"? Is there going to be another one? :smile:

Reply:
Wishful thinking that we were going to have significantly more of these tests.